app/marketing/views.py

```diff
@@ -272,6 +272,42 @@ def email_settings(request, key):
     email = ''
     level = ''
     msg = ''
+    email_types = {
+        'welcome_mail': 'Welcome Email',
```

Comment:
this already exists in `retail.emails.MARKETING_EMAILS`  - it should be DRYd up

Reply:
I don't understand this. Care to elaborate? `retail.emails.MARKETING_EMAILS.welcome_mail` you mean?

Reply:
DRY = dont' repeat yourself.  we dont need another `email_types` definition dict when it already exists on `retail.emails.MARKETING_EMAILS`

Reply:
```retail.emails.MARKETING_EMAILS``` is a list.

Reply:
it still contains the same information that you're re-declaring here and should be DRY'd..  mutate one data structure into the other if you need to